src/config/links.json

```
@@ -32,7 +32,7 @@
     "telegram": "#4995be",
     "tiktok": "#000000",
     "twitch": "#9146FF",
-    "twitter": "#00ACEE",
+    "twitter": "#1D9BF0",
```

The official color for Twitter is `rgb(29, 155, 240)` which is `#1d9bf0`. It should be 👇 
```suggestion
    "twitter": "#1d9bf0",
```